.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ version: 2.1
 # See https://blog.daemonl.com/2016/02/yaml.html
 # To validate changes, use an online parser, eg.
 # https://yaml-online-parser.appspot.com/
-var_1: &cache_key yarn-cache-{{ checksum "yarn.lock" }}-0.18.0
+var_1: &cache_key yarn-cache-{{ checksum "yarn.lock" }}-0.18.1
```

Comment:
Out of interest, why do we have the suffix 0.18.1? Is the checksum not already enough?

Reply:
The checksum should be enough, and was in this case not the culprit.
In the past we've had some experiences where the cache could go corrupt causing the CI build to act funky. That's why my first idea was to create a new cache.